server/py/framework/db/sqldb/db.py

```diff
@@ -5946,14 +5946,17 @@ def store_alert_activation(
                 session, alert_activation_record, "id"
             )
         else:
+            # for auto reset policy reset_time is the same as the activation time
+            # for manual reset policy, we keep it empty until policy reset
```

Comment:
```suggestion
            # for manual reset policy, we keep it empty until the alert is reset
```

Reply:
@TomerShor actually, that's alert being reset, reset policy just indicates when to reset it

Reply:
exactly, the alert is reset.
"until policy reset" means the the `policy` is reset, which is not the case.

Reply:
@TomerShor right, sorry :facepalm: